### PR TITLE
Item as kwarg

### PR DESCRIPTION
**Pull request by Licini**
Accommodating the changes from https://github.com/compas-dev/compas/pull/1353

### CODE REVIEW

File: src/compas_viewer/scene/collectionobject.py [hunk 1]
```diff
@@ -27,10 +27,14 @@ def __data__(self):
 class CollectionObject(ViewerSceneObject, GeometryObject):
     """Viewer scene object for displaying a collection of COMPAS geometries."""
 
-    def __init__(self, collection: Collection, **kwargs):
-        self.collection = collection
-        super().__init__(geometry=self.collection, **kwargs)
-        self.objects = [ViewerSceneObject(item, **kwargs) for item in self.collection.items]
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        kwargs.pop("item")
```
Comment (tomvanmele) [inline src/compas_viewer/scene/collectionobject.py:32]: what does this do?

File: src/compas_viewer/scene/groupobject.py [hunk 2]
```diff
@@ -18,8 +18,10 @@ def __data__(self):
 class GroupObject(SceneObject):
     """A group of scene objects."""
 
-    def __init__(self, items, **kwargs):
-        super().__init__(Group(items), **kwargs)
+    def __init__(self, **kwargs):
+        items = kwargs.pop("item", [])
```
Comment (tomvanmele) [inline src/compas_viewer/scene/groupobject.py:22]: why not just 
```python
def __init__(self, item=None, **kwargs):
    super().__init__(item=Group(item), **kwargs)

File: src/compas_viewer/scene/nurbscurveobject.py [hunk 3]
```diff
@@ -11,15 +11,10 @@
...
 
-    See Also
-    --------
-    :class:`compas.geometry.NurbsCurve`
-    """
-
-    def __init__(self, curve: NurbsCurve, **kwargs):
-        super().__init__(geometry=curve, **kwargs)
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
```
Comment (tomvanmele) [inline src/compas_viewer/scene/nurbscurveobject.py:17]: if this is only there to be able to set a type for `self.geometry`, you could also leave this away and set the annotation on a class attribute

```python
class NurbsCurveObject(...):
    geometry: NurbsCurve

File: src/compas_viewer/scene/polygonobject.py [hunk 4]
```diff
@@ -11,15 +11,10 @@
...
 
-    See Also
-    --------
-    :class:`compas.geometry.Polygon`
-    """
-
-    def __init__(self, polygon: Polygon, **kwargs):
-        super().__init__(geometry=polygon, **kwargs)
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
```
Comment (tomvanmele) [inline src/compas_viewer/scene/polygonobject.py:17]: same here

File: src/compas_viewer/scene/polyhedronobject.py [hunk 5]
```diff
@@ -10,15 +10,10 @@
...
 
-    See Also
-    --------
-    :class:`compas.geometry.Polyhedron`
-    """
-
-    def __init__(self, polyhedron: Polyhedron, **kwargs):
-        super().__init__(geometry=polyhedron, **kwargs)
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
```
Comment (tomvanmele) [inline src/compas_viewer/scene/polyhedronobject.py:16]: and here

File: src/compas_viewer/scene/groupobject.py (hunk 2 above)
Comment (Licini) [reply src/compas_viewer/scene/groupobject.py:22]: ha nice, didn't think of this way

File: src/compas_viewer/scene/collectionobject.py (hunk 1 above)
Comment (Licini) [reply src/compas_viewer/scene/collectionobject.py:32]: This was to remove item from kwargs so in next line `item` is not given twice which will trigger error.

**Comment by Licini** [comment]
@tomvanmele done, take another look?

File: src/compas_viewer/scene/collectionobject.py (hunk 1 above)
Comment (tomvanmele) [reply src/compas_viewer/scene/collectionobject.py:32]: just put item as a keyword argument in the first init

File: src/compas_viewer/scene/graphobject.py [hunk 6]
```diff
@@ -1,30 +1,11 @@
...
-    See Also
-    --------
-    :class:`compas.datastructures.Graph`
-
-    """
-
-    def __init__(self, graph: Graph, **kwargs):
-        super(GraphObject, self).__init__(graph=graph, **kwargs)
-        self.graph: Graph
+    """Viewer scene object for displaying COMPAS Graph data."""
```
Comment (tomvanmele) [inline src/compas_viewer/scene/graphobject.py:8]: why not add the type of graph

File: src/compas_viewer/scene/meshobject.py [hunk 7]
```diff
@@ -66,9 +64,7 @@ def __init__(
         use_vertexcolors: Optional[bool] = None,
         **kwargs,
     ):
-        super().__init__(mesh=mesh, **kwargs)
-
-        self.mesh: Mesh
+        super().__init__(**kwargs)
```
Comment (tomvanmele) [inline src/compas_viewer/scene/meshobject.py:67]: add the type of mesh as class attribute

**Review by tomvanmele — APPROVED** [review]
just a few small things. otherwise LGTM

**Pull request merged**